dakota_manylinux_install_files/build_deps_gha.sh

```diff
@@ -0,0 +1,176 @@
+patch -p1 < $CAROLINA_DIR/dakota_manylinux_install_files/DakotaFindPython.cmake.patch
+
+mkdir build
+cd build
+
+export PATH=/tmp/INSTALL_DIR/bin:$PATH
+export PYTHON_INCLUDE_DIRS="$PYTHON_INCLUDE_PATH $PYTHON_DEV_HEADERS_DIR /tmp/INSTALL_DIR/lib"
+export PYTHON_EXECUTABLE=$(which python)
+export LD_LIBRARY_PATH="$INSTALL_DIR/lib:/usr/local/lib:$PYTHON_INCLUDE_PATH:$NUMPY_INCLUDE_PATH:$NUMPY_INCLUDE_PATH/numpy:$PYTHON_DEV_HEADERS_DIR:/tmp/INSTALL_DIR/lib:$LD_LIBRARY_PATH"
+
+echo "export PATH=$PATH" >> /github/workspace/trace/env
+echo "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH" >> /github/workspace/trace/env
+echo "export PYTHON_INCLUDE_DIRS=$PYTHON_INCLUDE_DIRS" >> /github/workspace/trace/env
+echo "export PYTHON_EXECUTABLE=$PYTHON_EXECUTABLE" >> /github/workspace/trace/env
+
+export BOOST_PYTHON="boost_python$python_version_no_dots"
+export BOOST_ROOT=$INSTALL_DIR
+export PATH="$PATH:$INSTALL_DIR/bin"
+
+# More stable approach: Go via python
+numpy_lib_dir=$(find /tmp/myvenv/ -name numpy.libs)
+export LD_LIBRARY_PATH="/usr/lib:/usr/lib64:$INSTALL_DIR/lib:$INSTALL_DIR/bin:$numpy_lib_dir:$NUMPY_INCLUDE_PATH"
+export CMAKE_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed 's/::/:/g' | sed 's/:/;/g')
+export PYTHON_LIBRARIES="/usr/lib64/"
+export PYTHON_INCLUDE_DIR="/opt/_internal/cpython-3.7.17/include/python3.7m"
+export CMAKE_LINK_OPTS="-Wl,--copy-dt-needed-entries,-l pthread"
+
+echo "export BOOST_PYTHON=$BOOST_PYTHON" >> /github/workspace/trace/env
+echo "export BOOST_ROOT=$BOOST_ROOT" >> /github/workspace/trace/env
+echo "export PATH=$PATH" >> /github/workspace/trace/env
+echo "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH" >> /github/workspace/trace/env
+echo "export CMAKE_LIBRARY_PATH=\"$CMAKE_LIBRARY_PATH\"" >> /github/workspace/trace/env
+echo "export PYTHON_LIBRARIES=\"$PYTHON_LIBRARIES\"" >> /github/workspace/trace/env
+echo "export PYTHON_INCLUDE_DIR=\"$PYTHON_INCLUDE_DIR\"" >> /github/workspace/trace/env
+
+cmake_command="""
+cmake \
+      -DCMAKE_CXX_STANDARD=14 \
+      -DBUILD_SHARED_LIBS=ON \
+      -DDAKOTA_PYTHON=ON \
+      -DDAKOTA_PYTHON_DIRECT_INTERFACE=ON \
+      -DDAKOTA_PYTHON_DIRECT_INTERFACE_NUMPY=ON \
+      -DDAKOTA_DLL_API=OFF \
+      -DHAVE_X_GRAPHICS=OFF \
+      -DDAKOTA_ENABLE_TESTS=OFF \
+      -DDAKOTA_ENABLE_TPL_TESTS=OFF \
+      -DCMAKE_BUILD_TYPE="Release" \
+      -DDAKOTA_NO_FIND_TRILINOS:BOOL=TRUE \
+      -DCMAKE_INSTALL_PREFIX="$INSTALL_DIR" \
+      -DPYTHON_LIBRARIES=$PYTHON_LIBRARIES \
+      -DCMAKE_LINK_OPTIONS=\"$CMAKE_LINK_OPTS\" \
+      .. &> "$INSTALL_DIR/dakota_bootstrap.log"
+
+"""
+echo "# $cmake_command" >> /github/workspace/trace/env
+
+echo "Boostrapping Dakota ..."
+$($cmake_command &> /github/workspace/trace/dakota_bootstrap.log)
+
+echo "# make --debug=b -j8 install" >> /github/workspace/trace/env
+echo "Building Dakota ..."
+make --debug=b -j8 install &> /github/workspace/trace/dakota_install.log
+
+
+DIST_BUILD=/github/workspace/dist_build
```

Comment:
Maybe call it `DEPS_BUILD` instead of dist, as it is the directory with the dependencies you are copying to the install dir (which is closer to being the dist dir AFAIK)

Reply:
Let's do that, this makes sense. Fixing.